docs/ncu-ci.md

```diff
@@ -30,47 +30,160 @@ Options:
   --help      Show help                                                [boolean]
 ```
 
-## Example
+### `ncu-ci rate <type>`
 
-Get the CI results of PR 12345 (including latest results of each type of
-supported CI) and copy the summaries into clipboard:
+`ncu-ci rate <type>` calculate the success rate for CI jobs in the last 100 runs per [CI Health History](https://github.com/nodejs/reliability#ci-health-history), where `<type>` can be either `pr` for `node-test-pull-request` or `commit` for `node-test-commit`.
 
-```
-ncu-ci url https://github.com/nodejs/node/pull/12345 --copy
-```
-
-Get the results of job #12345 of  `node-test-pull-request`:
+Examples:
 
+```sh
+node on git:master ❯ ncu-ci rate pr
+--------------------------------------------------------------------------------
+[1/1] Running health
+--------------------------------------------------------------------------------
+✔  Done
+| UTC Time         | RUNNING | SUCCESS | UNSTABLE | ABORTED | FAILURE | Green Rate |
+| ---------------- | ------- | ------- | -------- | ------- | ------- | ---------- |
+| 2020-07-03 16:28 | 1       | 7       | 34       | 10      | 48      | 7.87%      |
 ```
-ncu-ci pr 12345
+
+```sh
+node on git:master ❯ ncu-ci rate commit
+--------------------------------------------------------------------------------
+[1/1] Running health
+--------------------------------------------------------------------------------
+✔  Done
+| UTC Time         | RUNNING | SUCCESS | UNSTABLE | ABORTED | FAILURE | Green Rate |
+| ---------------- | ------- | ------- | -------- | ------- | ------- | ---------- |
+| 2020-07-03 16:28 | 2       | 8       | 27       | 5       | 58      | 8.60%      |
 ```
 
+Possible use cases:
```

Comment:
Missing `ncu-ci walk` header?

Reply:
whoops, you're totally right - fixed and added an example!